ui/src/views/Documentation/Reference/ApiReference.jsx

```diff
@@ -9,6 +9,29 @@ import HeaderWithAnchor from '../components/HeaderWithAnchor';
 import Anchor from '../components/Anchor';
 import findRefDoc from '../../../utils/findRefDoc';
 
+class GroupEntry extends Component {
+  render() {
+    const { group, serviceName } = this.props;
+    const [groupName, listOfEntries] = group;
+
+    return (
+      <div>
+        <br />
+        <Typography> {groupName} </Typography>
```

Comment:
Use the header with anchor component instead to make it linkable :)

```jsx
<HeaderWithAnchor type="h4">{groupName}</HeaderWithAnchor>
```

Reply:
Applied the changes!